Source/Parallelization/WarpXRegrid.cpp

```diff
@@ -358,57 +282,14 @@ WarpX::RemakeLevel (int lev, Real /*time*/, const BoxArray& ba, const Distributi
         if (lev > 0 && (n_field_gather_buffer > 0 || n_current_deposition_buffer > 0)) {
             for (int idim=0; idim < 3; ++idim)
             {
-                if (Bfield_cax[lev][idim])
-                {
-                    const IntVect& ng = Bfield_cax[lev][idim]->nGrowVect();
-                    auto pmf = std::make_unique<MultiFab>(Bfield_cax[lev][idim]->boxArray(),
-                                                                      dm, Bfield_cax[lev][idim]->nComp(), ng);
-                    // pmf->ParallelCopy(*Bfield_cax[lev][idim], 0, 0, Bfield_cax[lev][idim]->nComp(), ng, ng);
-                    Bfield_cax[lev][idim] = std::move(pmf);
-                }
-                if (Efield_cax[lev][idim])
-                {
-                    const IntVect& ng = Efield_cax[lev][idim]->nGrowVect();
-                    auto pmf = std::make_unique<MultiFab>(Efield_cax[lev][idim]->boxArray(),
-                                                                      dm, Efield_cax[lev][idim]->nComp(), ng);
-                    // pmf->ParallelCopy(*Efield_cax[lev][idim], 0, 0, Efield_cax[lev][idim]->nComp(), ng, ng);
-                    Efield_cax[lev][idim] = std::move(pmf);
-                }
-                if (current_buf[lev][idim])
-                {
-                    const IntVect& ng = current_buf[lev][idim]->nGrowVect();
-                    auto pmf = std::make_unique<MultiFab>(current_buf[lev][idim]->boxArray(),
-                                                                      dm, current_buf[lev][idim]->nComp(), ng);
-                    // pmf->ParallelCopy(*current_buf[lev][idim], 0, 0, current_buf[lev][idim]->nComp(), ng, ng);
-                    current_buf[lev][idim] = std::move(pmf);
-                }
-            }
-            if (charge_buf[lev])
-            {
-                const IntVect& ng = charge_buf[lev]->nGrowVect();
-                auto pmf = std::make_unique<MultiFab>(charge_buf[lev]->boxArray(),
-                                                                  dm, charge_buf[lev]->nComp(), ng);
-                // pmf->ParallelCopy(*charge_buf[lev][idim], 0, 0, charge_buf[lev]->nComp(), ng, ng);
-                charge_buf[lev] = std::move(pmf);
-            }
-            if (current_buffer_masks[lev])
-            {
-                const IntVect& ng = current_buffer_masks[lev]->nGrowVect();
-                auto pmf = std::make_unique<iMultiFab>(current_buffer_masks[lev]->boxArray(),
-                                                                    dm, current_buffer_masks[lev]->nComp(), ng);
-                // we can avoid this since we immediately re-build the values via BuildBufferMasks()
```

Comment:
Is it worth copying this comment over, and inserting 'redistributing' before 'this'?